src/cq_cam/routers.py

```diff
@@ -264,10 +321,17 @@ def shift_polygon(polygon: Path, i: int):
     return polygon
 
 
-def route_polyface_outers(job: "Job", polyfaces: list[PathFace], stepover=None):
+def route_polyface_outers(
+    job: "Job",
+    polyfaces: list[PathFace],
+    stepover=None,
+    start_cv: CommandVector = CommandVector(),
+) -> list[MotionCommand]:
     commands = []
     previous_wire_end = None
 
+    start_cv = CommandVector()
```

Comment:
Overrides the function argument.

Reply:
Is CommandVector used by the way to enable passing out state from the function using mutability? I'd rather have the function to just return a tuple. Mutable side effects, while sometimes useful, are more often one cause of premature hair loss.

Reply:
start_cv is not used to pass state out but rather to pass it in. It was also the source of the bug I reported. It currently has been fixed but I need to do a proper cleanup (hence the conversion of the PR into a draft).